dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGVisualizer.java

```diff
@@ -48,68 +49,76 @@
      * call back to this visualizer instance for sub-components.
      *
      * @param store the store to visualize
+     * @return the String presentation of store
      */
-    void visualizeStore(S store);
+    String visualizeStore(S store);
 
     /**
      * Called by a {@code CFAbstractStore} to visualize the class name before calling the {@code
      * CFAbstractStore#internalVisualize()} method.
      *
      * @param classCanonicalName the canonical name of the class
+     * @return the String representation of the header of store
```

Comment:
When I read this comment, I was unclear what store this is referring to.  This method doesn't take a store as an argument.
I think the answer is that this class contains a "current store".  `visualizeStore` sets it, and other methods implicitly use it.
I suggest that this should be documented in the class documentation and in `visualizeStore`, and this comment should refer to the "current store".
It's a bit confusing that the class documentation says that the class contains a CFG but doesn't say anything about it maintaining a current store.

I realize this confusion about the design isn't your fault and I don't wish to block the pull request on a lot of extra work.  However, you probably understand the design now and can make some clarifying comments along with this pull request.

Thanks!

Reply:
There is no current store that is maintained by the CFGVisualizers. This method only returns the "header" for a store, which is only based on the passed class name.
The method description explains this correctly, I think.
So how about changing the return description to:
`@return the header to use for a store with the given canonical name`?

Reply:
> There is no current store that is maintained by the CFGVisualizers. This method only returns the "header" for a store, which is only based on the passed class name.
> The method description explains this correctly, I think.
> So how about changing the return description to:
> `@return the header to use for a store with the given canonical name`?

That sounds good to me.